UnityProject/Assets/Scripts/Player/PlayerParticle.cs

```diff
@@ -47,16 +47,18 @@ public void ServerToggleParticle(string id, bool state)
 				}
 				return;
 			}
+			netIdentity.isDirty = true;
 			Loggy.LogError($"[PlayerParticle/ToggleParticle] - no such object named {id}.");
 		}
 
-		public void OnActiveIdListChange(List<string> oldState, List<string> newState)
+		public void OnActiveIdListChange(SyncList<string> oldState, SyncList<string> newState)
 		{
 			if (particles == null || particles.Count == 0) return;
 			foreach (var particle in particles)
 			{
 				particle.Reference.SetActive(newState.Contains(particle.ID));
 			}
+			netIdentity.isDirty = true;
```

Comment:
you shouldn't need to do this?

Reply:
the update state for synclists is a bit more complicated than this

Reply:
https://mirror-networking.gitbook.io/docs/manual/guides/synchronization/synclists

Reply:
> you shouldn't need to do this?

![image](https://github.com/unitystation/unitystation/assets/34368774/6dedb872-9378-466e-a089-5ec80855330e)

Reply:
well Need to work out how to modify Preprocessor , since it does some magic stuff to link up the synclist

Reply:
Converted them to use synclist operations